### PR TITLE
Feature/proxy get route

### PR DESCRIPTION
**Pull request by NFabrizio**

### CODE REVIEW

File: modelservice/master.py [hunk 1]
```diff
@@ -90,8 +90,31 @@ def BuildModels(self, request: BuildModelsRequest, context) -> BuildModelsRespon
         )
 
     def GetModel(self, request: GetModelRequest, context) -> GetModelResponse:
-        info(f"Received request to retrieve model(s)")
-        return GetModelResponse()
+        info(f"Received request to retrieve model")
+
+        # job_id: str = request.model_id
+        gis_join: str = request.gis_joins
```
Comment (inf0rmatiker) [inline modelservice/master.py]: Can you update this to `request.gis_join` (singular) in the proto file and regenerate the protobuf/grpc files

File: modelservice/master.py [hunk 2]
```diff
@@ -90,8 +90,31 @@ def BuildModels(self, request: BuildModelsRequest, context) -> BuildModelsRespon
...
+                filename="",
+                data=""
+            )
+
+        info(f"Launching get worker model for {worker.hostname}...")
+        with grpc.insecure_channel(f"{worker.hostname}:{worker.port}") as channel:
+            stub = modelservice_pb2_grpc.WorkerStub(channel)
+            # TODO: Determine if we need to make a copy when we just do a pass through
+            request_copy = GetModelRequest()
+            request_copy.CopyFrom(request)
```
Comment (inf0rmatiker) [inline modelservice/master.py]: No need to copy request here you can use a reference, we're not changing anything in it

File: modelservice/proxy.py [hunk 3]
```diff
@@ -34,14 +35,36 @@ def run(master_hostname="localhost", master_port=50051, flask_port=5000):
     app.run(host="0.0.0.0", port=flask_port)  # Entrypoint
 
 
-@app.route("/model", methods=["GET"])
-def get_model():
+@app.route("/model/<model_id>/gis-join/<gis_join>", methods=["GET"])
```
Comment (inf0rmatiker) [inline modelservice/proxy.py:38]: This is great I love it

File: modelservice/proxy.py [hunk 4]
```diff
@@ -34,14 +35,36 @@ def run(master_hostname="localhost", master_port=50051, flask_port=5000):
...
+def get_model(model_id, gis_join):
 
-    # TODO Parse query from HTTP JSON -> build gRPC request to Master
+    # Try to cast request data to proper types and return error if there are any issues
+    try:
+        get_model_grpc_request: GetModelRequest = GetModelRequest(
+            model_id=model_id,
+            gis_joins=gis_join
```
Comment (inf0rmatiker) [inline modelservice/proxy.py]: Lol same thing here. This will have to be updated if you make gis_joins singular in the gRPC message

Comment (NFabrizio) [reply modelservice/proxy.py]: Updated

File: modelservice/master.py (hunk 2 above)
Comment (NFabrizio) [reply modelservice/master.py]: Removed

File: modelservice/master.py (hunk 1 above)
Comment (NFabrizio) [reply modelservice/master.py]: Updated

**Pull request merged**